src/idstore.js

```diff
@@ -1,18 +1,17 @@
 'use strict'
 
-const Block = require('ipld-block')
 const filter = require('it-filter')
-const mh = require('multihashes')
 const pushable = require('it-pushable')
 const drain = require('it-drain')
-const CID = require('cids')
+const { CID } = require('multiformats')
```

Comment:
you should be able to `require('multiformats/cid')` here for the same reason mentioned above, avoid the root export if/where possible if you only want one thing

Reply:
```suggestion
const { CID } = require('multiformats/cid')
```